### PR TITLE
[Ide] Add explanation in help on how to open file at line and col

### PR DESCRIPTION
**Pull request by Therzok**
Bug 9428 - open to line from terminal command line

### CODE REVIEW

**Comment by Therzok** [comment]
Couldn't find a way to do it from Mono.Options, so I just manually printed it out.

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/IdeStartup.cs [hunk 1]
```diff
@@ -731,6 +731,10 @@ public static MonoDevelopOptions Parse (string[] args)
 				Console.WriteLine (BrandingService.ApplicationName + " " + BuildInfo.VersionLabel);
 				Console.WriteLine ("Options:");
 				optSet.WriteOptionDescriptions (Console.Out);
+				const string openFileText = "      file.ext;line;column";
+				Console.Write (openFileText);
+				Console.Write (new string (' ', 29 - openFileText.Length));
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/IdeStartup.cs]: This is a magic number from Mono.Options.

**Comment by slluis** [comment]
LGTM

**Comment by Therzok** [comment]
Manually merged in 1f15d81

**Pull request closed without merging**